tests/testthat/test-independent-testing-tlf-ae-listing.R

```diff
@@ -0,0 +1,28 @@
+library(metalite)
```

Comment:
Please 

1. create a folder under `tests/testthat` and name this new folder `fixtures`
2. move the `test_meta_specific.rda` into `fixtures`
3. remove the folder `data`.